pyomo/core/expr/numeric_expr.py

```diff
@@ -567,27 +552,7 @@ class NPV_DivisionExpression(NPV_Mixin, DivisionExpression):
     __slots__ = ()
 
 
-class _LinearOperatorExpression(NumericExpression):
```

Comment:
I'm not opposed to removing this class, but I'm curious what motivated it.

Reply:
It was only used by `SumExpressionBase` (because it is not longer used by the relational expressions). Having the "base class" logic split across two classes just added unnecessary overhead.  Especially since not all linear operators were using `_LinearOperatorExpression`